addons/crm/static/src/views/crm_kanban/crm_kanban_model.js

```diff
@@ -6,7 +6,7 @@ import { checkRainbowmanMessage } from "@crm/views/check_rainbowman_message";
 export class CrmKanbanModel extends KanbanModel {
     setup(params, { orm, effect }) {
         super.setup(...arguments);
-        this.ormService = orm;
+        this.ormService = orm; // why not using the ormBatcher ?
```

Comment:
We definitely can, most likely I didn't pay attention and didn't realise we already had access to the orm from KanbanModel, we can also remove "orm" from the services down this file